server/auth/routes.py

```diff
@@ -93,7 +93,12 @@ def auth_me():
                 for org in user.organizations
             ],
             jurisdictions=[
-                {"id": j.id, "name": j.name, "election": serialize_election(j.election)}
+                {
+                    "id": j.id,
+                    "name": j.name,
+                    "election": serialize_election(j.election),
+                    "numBallots": j.manifest_num_ballots,
```

Comment:
ahah, I see, this is where you're copying the `manifest.numBallots` to `jurisdiction.numBallots`, but still want to make sure you're confident in using both, as per my earlier comment.